backend/app/repository/gerrit_provider.py

```
@@ -0,0 +1,928 @@
+            git_info = self._pick_git_info(user, git_domain)
+            git_domain = git_info["git_domain"]
+
+        cache_key = cache_manager.generate_full_cache_key(user.id, git_domain)
+        return await cache_manager.get(cache_key)
+
+    async def get_branch_diff(
+        self,
+        user: User,
+        repo_name: str,
+        source_branch: str,
+        target_branch: str,
```

Ruff (0.14.5)

Unused method argument: `target_branch`

(ARG002)